ext/openssl/openssl.c

```diff
@@ -7642,12 +7642,8 @@ PHP_FUNCTION(openssl_cipher_iv_length)
 PHP_OPENSSL_API zend_string* php_openssl_random_pseudo_bytes(zend_long buffer_length)
 {
 	zend_string *buffer = NULL;
-	if (buffer_length <= 0
-#ifndef PHP_WIN32
-		|| ZEND_LONG_INT_OVFL(buffer_length)
-#endif
-			) {
-		zend_argument_value_error(1, "must be greater than 0");
+	if (buffer_length <= 0 || ZEND_LONG_INT_OVFL(buffer_length)) {
+		zend_argument_value_error(1, "must be greater than 0 and less than 2147483648");
```

Comment:
Could your rather split it to two condition and different error message for each so we don't change the current message?

Reply:
Done, although I doubt that changing the error message would be a real BC problem; after all, this is a `ValueError`.